### DISCUSSION TITLE
Discussion: Do you use any other timer app besides Safe Eyes?

### DISCUSSION THREAD

**Discussion by orschiro** [slgobinath/safeeyes#609] [General]
Hi all, 

Do we have a discussion forum? I don't think we do, correct?

If so, please apologise having posted this here.

My question: 

Do you guys combine SafeEyes with another timer such as a Pomodoro work schedule, for example?

**Comment by slgobinath** [comment]
Hi @orschiro,

We don't have a forum so I've created a label `discussion` and marked this as a discussion. I hope this is much easier than maintaining a forum.

In my case, I don't use any other timers.

**Comment by orschiro** [comment]
Thanks!

Me neither. 

I am just wondering whether others see any merit in running another timer.

**Comment by orschiro** [comment]
An idea I just had based on our conversation:

Maybe Safe Eyes could display a small note after having completed one entire cycle of breaks to make the user aware of how many of those cycles or sessions he is actually performing throughout the day?

Perhaps an interesting statistics. :-)

**Comment by slgobinath** [comment]
I had a different statistics in mind. How many breaks the user has taken and/or skipped, how long he/she is using the computer in a day, etc as an overall health report. I'll add the number of cycles into that statistic.

**Comment by orschiro** [comment]
Cool, thanks!

**Comment by ruXlab** [comment]
Nothing apart of google calendar integrated to unity.  
usecase: developer

**Comment by orschiro** [comment]
@ruXlab thanks! Although I wouldn't really consider Google Calendar a timer app. :-D

How do you integrate it into Unity?

**Comment by ruXlab** [comment]
@orschiro using evolution, https://askubuntu.com/questions/613589/google-calendar-in-unity

**Comment by radek-sprta** [comment]
No, not at all.

**Comment by anarcat** [comment]
i'm currently using workrave, which has the nice feature of counting the number of clicks and measuring mouse movement, out of which i make report every once in a while:

https://anarc.at/blog/2016-06-01-work-volume/

it'd be nice to see safeeyes support this as well, see #184.

**Comment by FedericoCeratto** [comment]
For keystroke metrics https://github.com/gurgeh/selfspy comes to mind. ![Debian RFP #873955](https://btsbadges.debian.net/badges/wnpp/selfspy)

**Comment by slgobinath** [comment]
Hi, all,
This comment is to make some noise about Safe Eyes 2.0.0 https://github.com/slgobinath/SafeEyes/issues/189 but still, I guess it makes sense for this discussion. I kindly request you to test Safe Eyes 2.0.0 (currently under development but almost ready) and provide your feedback.

In addition, using the new plugin architecture, one can easily use the advantage of running Safe Eyes in some other contexts. For example, the `next to do item` or `current weather` can be displayed on the Safe Eyes break screen if suitable plugins are developed. I have already developed a `weather` plugin. Soon I will push it to the repository.

A `Health Statistics` plugin is already there with minimal features. I am looking forward to improve it further.

**Comment by anarcat** [comment]
@FedericoCeratto awesome - i'm using selfspy now! :)

**Comment by Hritik14** [comment]
I use i3 and have created a block which uses `safeeyes --status` and `date` to figure out time remaining for next break and shows a timer in the bar. I've disabled all short breaks and all long breaks occur at 25 min, effectively using the pomodoro technique.

**Comment by FedericoCeratto** [comment]
I use an external pomodoro timer because the screen locking is too intrusive - see #275

**Comment by wolftune** [comment]
Since https://github.com/slgobinath/SafeEyes/discussions exists now, does it make sense to close this and move it there (or can it be moved?)

Anyway, I was just comparing options and found that https://github.com/hovancik/stretchly/ and https://github.com/rcaelers/workrave/ are the two most compelling other options. While testing, I tried multiple at once, but I know that's not the normal use of multiple-timer-apps. I found Workrave special because it has custom pre-break prompts that show on multiple-screens and then do not start the break until system input is idle. That's much better than mere system-notification warnings. I otherwise like SafeEyes, and Stretchly is almost the same (but is a heavier electron app).

**Comment by nifadyev** [comment]
Hey @AdamPS , shall this issue be converted in Github Discussion?

**Comment by FedericoCeratto** [comment]
I'm also using https://timewarrior.net/
